examples/notebooks/knn_mnist.ipynb

```diff
@@ -244,6 +253,52 @@
     "print(f\"Classification report for KNN:\\n{report}\\n\")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "877119db",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<style type=\"text/css\">\n",
+       "</style>\n",
+       "<table id=\"T_51c5f_\">\n",
+       "  <thead>\n",
+       "    <tr>\n",
+       "      <th class=\"col_heading level0 col0\" >Intel(R) extension for Scikit-learn</th>\n",
+       "      <th class=\"col_heading level0 col1\" >Original Scikit-learn</th>\n",
+       "      <th class=\"col_heading level0 col2\" >Difference</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td id=\"T_51c5f_row0_col0\" class=\"data row0 col0\" >2.516654</td>\n",
+       "      <td id=\"T_51c5f_row0_col1\" class=\"data row0 col1\" >21.113301</td>\n",
+       "      <td id=\"T_51c5f_row0_col2\" class=\"data row0 col2\" >8.389435</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n"
+      ],
+      "text/plain": [
+       "<pandas.io.formats.style.Styler at 0x7fd5cbdd43d0>"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
```

Comment:
I think better without pandas. Add to summary maybe. @SmirnovEgorRu what do you think?

Reply:
@SmirnovEgorRu, What do you think about pandas table in examples?
https://github.com/Pahandrovich/daal4py/blob/fix-examples/examples/notebooks/knn_mnist.ipynb